packager/media/codecs/avc_decoder_configuration_record_unittest.cc

```diff
@@ -89,9 +90,10 @@ TEST(AVCDecoderConfigurationRecordTest, SuccessWithSPSExtension) {
   EXPECT_EQ(8u, avc_config.pixel_width());
   EXPECT_EQ(9u, avc_config.pixel_height());
   EXPECT_EQ(0u, avc_config.transfer_characteristics());
-
-  EXPECT_EQ("avc1.64001e", avc_config.GetCodecString(FOURCC_avc1));
-  EXPECT_EQ("avc3.64001e", avc_config.GetCodecString(FOURCC_avc3));
+  EXPECT_EQ(0u, avc_config.transfer_characteristics());
```

Comment:
This is duplicated with the line above it. Please remove.

Reply:
done